### PR TITLE
Feature#55/report views

### LINKED ISSUE #55: reports view
# REQUIREMENTS
- Codify the designs proposed by the team.
- Use SASS for the styles in the code.
- Adapt as much as possible to the design. 
- Create the views and their respective routes.

# DESIGN
image
image
image

# Acceptance Criteria
List all required acceptance criteria
- [x] SASS Tools
- [x] Path of the views
- [x] Layout adaptation.

### PR DESCRIPTION
**Pull request by jairolan**
# Description
What did you implemented/Why did you implemented this:
- I have generated the controller and report view. 
- I have configured the routes using get methods for now.
- I have written the css styles for the main report views.
 
 # Testing
 Help me how can I test or look at the changes
 
  ## Screenshots
image
image
image

### CODE REVIEW

File: reloj-checador/app/controllers/reports_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ReportsController < ApplicationController
+    def index; end
+
```
Comment (juliocabrera820) [inline reloj-checador/app/controllers/reports_controller.rb]: delete blank line

File: reloj-checador/app/controllers/reports_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class ReportsController < ApplicationController
+    def index; end
+    
+    def daily_attendance; end
+    def monthly_attendance; end
+    def monthly_absence; end
+    def average_time; end
+
```
Comment (juliocabrera820) [inline reloj-checador/app/controllers/reports_controller.rb]: delete blank line

File: reloj-checador/app/helpers/reports_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module ReportsHelper
+end
```
Comment (juliocabrera820) [inline reloj-checador/app/helpers/reports_helper.rb]: remove helper

File: reloj-checador/app/javascript/stylesheets/application.scss [hunk 4]
```diff
@@ -1,4 +1,7 @@
 @import 'dashboard';
 @import 'menu_admin';
 @import 'modal-side';
-@import 'devise';
\ No newline at end of file
+@import 'devise';
+@import 'reports';
+@import 'attendance_styles';
```
Comment (juliocabrera820) [inline reloj-checador/app/javascript/stylesheets/application.scss]: you can keep the name without suffix styles

File: reloj-checador/app/javascript/stylesheets/attendance_styles.scss [hunk 5]
```diff
@@ -0,0 +1,69 @@
+
+  .layout{
+	  margin-top: 40px;
```
Comment (juliocabrera820) [inline reloj-checador/app/javascript/stylesheets/attendance_styles.scss]: keep your indentation

File: reloj-checador/app/javascript/stylesheets/average.scss [hunk 6]
```diff
@@ -0,0 +1,70 @@
+
+  .average-layout{
+	margin-top: 50px;
```
Comment (juliocabrera820) [inline reloj-checador/app/javascript/stylesheets/average.scss]: keep your indentation

File: reloj-checador/app/views/reports/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<h1 class="title-reports">Reports</h1>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm">
+        <div class="card pointer">
+          <div class="container">
+            <h4 class="text-layout"> <%= link_to "Daily attendance", reports_daily_attendance_path, class: "link-layout" %></h4>
+
```
Comment (juliocabrera820) [inline reloj-checador/app/views/reports/index.html.erb]: delete blank line

File: reloj-checador/app/views/reports/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      </div>
+      <div class="col-sm">
+				<div class="card pointer">
+          <div class="container">
+            <h4 class="text-layout"><%= link_to "Monthly absence", reports_monthly_absence_path, class: "link-layout" %></h4>
+          </div>
+        </div> 
+      </div>
+    </div>
+	</div>
```
Comment (juliocabrera820) [inline reloj-checador/app/views/reports/index.html.erb:31]: keep your indentation

**Review by juliocabrera820 — DISMISSED** [review]
Nice job

File: reloj-checador/app/javascript/stylesheets/attendance.scss [hunk 9]
```diff
@@ -0,0 +1,63 @@
+  .layout{
+	  margin-top: 40px;
```
Comment (jaimelr) [inline reloj-checador/app/javascript/stylesheets/attendance.scss:2]: Check indentation.

File: reloj-checador/app/javascript/stylesheets/attendance.scss [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    margin-left: 150px;
+    color: #583D72;
+    font-weight: bolder;
+    font-size: 18px;
+  }
+  .save{
```
Comment (jaimelr) [inline reloj-checador/app/javascript/stylesheets/attendance.scss]: Space before ` {`, check the entire file for this type of issues.

File: reloj-checador/app/views/reports/average_time.html.erb [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+  <div class="container">
+    <h5 class="average-title-check">Check out</h5>
+    <div class="row">
+      <div class="col-sm">
+        <p class="average-layout">Week of the day</p> 
+      </div>  
+      <div class="col-sm">
+        <div class="average-dropdown">
+          <button class="average-dropbtn">Dropdown</button>
+            <div class="average-dropdown-content">
```
Comment (jaimelr) [inline reloj-checador/app/views/reports/average_time.html.erb:54]: If this is a dropdown I suggest using Rails `select_tag` https://guides.rubyonrails.org/v2.3/form_helpers.html#the-select-and-option-tags

If this is just a placeholder and you plan on removing it later, then ignore.

File: reloj-checador/app/views/reports/index.html.erb [hunk 12]
```diff
@@ -0,0 +1,32 @@
+<h1 class="title-reports">Reports</h1>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm">
+        <div class="card pointer">
+          <div class="container">
+            <h4 class="text-layout"> <%= link_to "Daily attendance", reports_daily_attendance_path, class: "link-layout" %></h4> 
+          </div>
+        </div>
+				<div class="card pointer">
```
Comment (jaimelr) [inline reloj-checador/app/views/reports/index.html.erb:10]: Fix indentation in this file, please.

**Review by jaimelr — DISMISSED** [review]
Looks good overall. I left a couple of non-blocking comments.

File: reloj-checador/app/javascript/stylesheets/attendance.scss (hunk 9 above)
Comment (jairolan) [reply reloj-checador/app/javascript/stylesheets/attendance.scss:2]: @jaimelr I have an error when my code is uploaded, locally my identation is correct.
image

File: reloj-checador/app/views/reports/average_time.html.erb (hunk 11 above)
Comment (jairolan) [reply reloj-checador/app/views/reports/average_time.html.erb:54]: Thanks

**Review by MartinSGL — APPROVED** [review]

**Review by juliocabrera820 — APPROVED** [review]
LGTM 👍🏼

**Review by MartinSGL — APPROVED** [review]

**Review by oramasangel — APPROVED** [review]
LGTM

**Pull request merged**